### PR TITLE
added tracking snippet for clarity analytics

### PR DESCRIPTION
**Pull request by manolingam**

### CODE REVIEW

File: packages/web/pages/_app.tsx [hunk 1]
```diff
@@ -39,6 +39,19 @@ const App: React.FC<WithUrqlProps> = ({
           />
         </>
       )}
+      {CONFIG.clarityID != null && (
```
Comment (dan13ram) [inline packages/web/pages/_app.tsx:42]:
```suggestion
      {!!CONFIG.clarityID && (
```
would this be cleaner here ?

**Review by dan13ram — CHANGES_REQUESTED** [review]
Could you please rebase this PR with the latest `develop` branch ?

File: packages/web/pages/_app.tsx (hunk 1 above)
Comment (manolingam) [reply packages/web/pages/_app.tsx:42]: Yeah, right.

**Comment by manolingam** [comment]
> Could you please rebase this PR with the latest `develop` branch ?

Why not merge?

File: packages/web/pages/_app.tsx (hunk 1 above)
Comment (dan13ram) [reply packages/web/pages/_app.tsx]: > Why not merge?

Rebase is the dev process we follow @manolingam  :)
With rebase only 1/2 commits of yours would be in this PR

Comment (manolingam) [reply packages/web/pages/_app.tsx]: Just updated.

**Comment by dan13ram** [comment]
> Why not merge?

Rebase is the dev process we follow @manolingam  :)
With rebase only 1/2 commits of yours would be in this PR

**Comment by manolingam** [comment]
> > Why not merge?
> 
> Rebase is the dev process we follow @manolingam :)
> With rebase only 1/2 commits of yours would be in this PR

I will make a PR from `develop` then.

**Comment by manolingam** [comment]
> > Why not merge?
> 
> Rebase is the dev process we follow @manolingam :)
> With rebase only 1/2 commits of yours would be in this PR

Just did a force rebase.

**Comment by manolingam** [comment]
Will make a new PR.

**Pull request closed without merging**